pkgs/development/libraries/ldacbt/default.nix

```diff
@@ -30,7 +30,7 @@ stdenv.mkDerivation rec {
     description = "AOSP libldac dispatcher";
     homepage    = "https://github.com/EHfive/ldacBT";
     license     = licenses.asl20;
-    platforms   = platforms.all;
+    platforms   = platforms.littleEndian;
```

Comment:
Please add a comment clarifying the situation.

Reply:
Comment added:

```nix
# libldac code detects & #error's out on non-LE byte order
```